### DISCUSSION TITLE
How can I use multiple O3d cameras or other ifm products to measure very large objects volume(such as a plane)? Is there a similar off-the-shelf application? Or how am I going to develop this software?

### DISCUSSION THREAD

**Discussion by mikufans-dumo** [ifm/ifm3d#459] [Q&A]
How can I use multiple O3d cameras or other ifm products to measure very large objects volume(such as a plane)?
 Is there a similar off-the-shelf application? 
Or how am I going to develop this software?

I just got my first TOF camera（O3D312） last week，I Completely unknown how to do。。。。

**Comment by lola-masson** [comment]
Hi @mikufans-dumo,

The O3D312 is the "sensor" version, which comes with some predefined applications that you can use out of the box. 
Currently, we do not provide an application which would use multiple cameras together for dimensioning, you would probably need to develop the code for this yourself.

To best assist you, I would recommend reaching out over email to discuss your application. Would you mind sharing which country you are located in? This will help us direct you to the relevant person to have this discussion.
